django_projects/locallibrary/catalog/admin.py

```diff
@@ -1,3 +1,31 @@
 from django.contrib import admin
+from .models import Author,Genre,Book,BookInstance
 
-# Register your models here.
+class AuthorAdmin(admin.ModelAdmin):
```

Comment:
Dùng đồng nhất 1 cách register site admin thôi em

VD: Line 4 -> line 8
```suggestion
@admin.register(Author)
class AuthorAdmin(admin.ModelAdmin):
    list_display = ('last_name', 'first_name', 'date_of_birth', 'date_of_death')
    fields = ['first_name', 'last_name', ('date_of_birth', 'date_of_death')]
```